tests/setup_conductor.js

```diff
@@ -23,9 +23,18 @@ function extract_sanitized_lines ( buf ) {
 async function start_conductor () {
     if ( holochain === undefined ) {
 	const cmd			= "holochain";
-	const args			=  "-c conductor-1.toml".split(" "); //  > conductor.log 2>&1 & tail -f conductor.log
+	const args			=  "-c ./tests/condutor-storage/conductor-config.yml".split(" "); //  > conductor.log 2>&1 & tail -f conductor.log
+	const opts		=  {
+		stdio: "inherit",
+		env: {
+		  ...process.env,
+		  RUST_LOG: process.env.RUST_LOG ? process.env.RUST_LOG : "info",
+		},
+	  }
+
 	log.debug("Spawning child holochain: %s %s (pwd %s)", cmd, args, process.env.PWD );
-	holochain			= ChildProcess.spawn( cmd, args );
+	
+	holochain			= ChildProcess.spawn( cmd, args, opts );
```

Comment:
We are running conductor using make.  I think we eventually want to spin them up using node, but this takes a lot more code to do properly.  I don't see any code for handling graceful shutdowns here.